client/components/inventory.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+var Inventory = (props) => {
```

Comment:
Something you can do to reduce the typing of 'props' is destructure them,
Rather than passing in props as the argument you can write something like
var Inventory = ( { count, size } ) => ...